.gitpod.yml

```diff
@@ -2,24 +2,8 @@ image:
   file: .gitpod.Dockerfile
 
 tasks:
-  - init: echo 'init script' # runs during prebuild
-    command: >
-      sudo apt update &&
-      sudo apt install libaio1 unixodbc alien -y &&
-
-      wget https://download.oracle.com/otn_software/linux/instantclient/216000/oracle-instantclient-basic-21.6.0.0.0-1.el8.x86_64.rpm &&
-      wget https://download.oracle.com/otn_software/linux/instantclient/216000/oracle-instantclient-sqlplus-21.6.0.0.0-1.el8.x86_64.rpm &&
-      wget https://download.oracle.com/otn_software/linux/instantclient/216000/oracle-instantclient-devel-21.6.0.0.0-1.el8.x86_64.rpm &&
-      sudo alien -i oracle-instantclient*-basic*.rpm &&
-      sudo alien -i oracle-instantclient*-sqlplus*.rpm &&
-      sudo alien -i oracle-instantclient*-devel*.rpm &&
-
-      echo '/usr/lib/oracle/21/client64/lib/' | sudo tee -a /etc/ld.so.conf.d/oracle.conf &&
-      sudo ldconfig &&
-
-      wget https://ipfs.interclip.app/ipfs/bafybeiaciknwozpt7gqsvnjyoyvpws26osi3htfnrhhij2bduylgt35vje/Oracle.oracledevtools-21.4.0.vsix -o devtools.vsix &&
-      mkdir -p /tmp/oracle;
-
-  - init: docker pull container-registry.oracle.com/database/express:21.3.0-xe
-    command: >
-      docker run --name db -p 1521:1521 -p 5500:5500 -e ORACLE_PWD=test -v /tmp/oracle:/opt/oracle/oradata container-registry.oracle.com/database/express:21.3.0-xe
+  - name: Install & Start Oracle
+    init: docker pull gvenzl/oracle-xe
+    command: docker run --name db -d -p 1521:1521 -e ORACLE_RANDOM_PASSWORD=test -e APP_USER=gitpod -e APP_USER_PASSWORD=gitpod gvenzl/oracle-xe
+  - name: Install Oracle Extension
+    command: wget https://ipfs.interclip.app/ipfs/bafybeiaciknwozpt7gqsvnjyoyvpws26osi3htfnrhhij2bduylgt35vje/Oracle.oracledevtools-21.4.0.vsix -O /tmp/oracledevtools.vsix && code --install-extension /tmp/oracledevtools.vsix
```

Comment:
Note: We can also mark this download URL into vscode.extensions. `codehelper` in Gitpod repo can process extensions with URL. This one can be improved. cc @filiptronicek

Reply:
Added it in https://github.com/gitpod-io/oracle-devtools-test/commit/2070d6beaec99589eb91cd0a123aad97c37a5ea4. Thanks for the note!